tests/Unit/inc/ThirdParty/Plugins/SimpleCustomCss/cacheSccss.php

```diff
@@ -19,25 +19,17 @@ public function setUp() {
 		parent::setUp();
 		$this->sccss = new SimpleCustomCss();
 
-		Functions\expect( 'rocket_has_constant' )->with( 'FS_CHMOD_DIR' )->andReturn( true );
-		Functions\expect( 'rocket_get_constant' )->with( 'FS_CHMOD_DIR' )->andReturn( 0755 & ~ umask() );
+		Functions\expect( 'rocket_get_constant' )->with( 'WP_ROCKET_CACHE_BUSTING_PATH' )->andReturn( 'wp-content/cache/busting/' );
+		Functions\expect( 'rocket_get_constant' )->with( 'WP_ROCKET_CACHE_BUSTING_URL' )->andReturn( 'http://example.org/wp-content/cache/busting/' );
```

Comment:
Just  a suggestion, this also can be written like this:
```suggestion
		Functions\expect( 'rocket_get_constant' )
		    ->with( 'WP_ROCKET_CACHE_BUSTING_PATH' )
		    ->andReturn( 'wp-content/cache/busting/' )
		    ->andAlsoExpectIt()
		    ->with( 'WP_ROCKET_CACHE_BUSTING_URL' )
		    ->andReturn( 'http://example.org/wp-content/cache/busting/' );
```